### DISCUSSION TITLE
Some fields are missing when form is submitted.

### DISCUSSION THREAD

**Discussion by paseto** [symfony/symfony#43530] [Q&A] [answered]
I have a parent form that allows users to add as many collection child entities as the user wants.
When a user adds more than 30 child forms, the next ones are not being sent to post.
Is there any limitation to submit a form?

![screenshot-127 0 0 1_8000-2021 10 15-10_32_16](https://user-images.githubusercontent.com/1163778/137496033-5b546b60-2bbe-475e-836e-83d3b60359bf.png)

**Comment by stof** [comment, marked as answer]
PHP has a limit of the size of `$_POST`. By default, it is 1000 values being parsed (that's configurable in php.ini). So depending on the size of your subforms, that might explain it.

**Reply by paseto** [reply]
You're right! Thank you @stof!

This PHP function it's something I would never imagine change someday.

PS.: If anyone has the same issue, after restart the webserver, should restart Symfony as well.
